File: app/src/main/java/com/otraempresa/android/example/RequestMerchantSettingsTask.java
package com.otraempresa.android.example;

import android.content.Context;

import com.kushkipagos.android.KushkiException;
import com.kushkipagos.android.MerchantSettings;

public class RequestMerchantSettingsTask extends AbstractRequestMerchantSettingsTask {
    RequestMerchantSettingsTask(Context context){
        super(context);
    }

    @Override
    protected MerchantSettings requestMerchantSettings() throws KushkiException{
        return kushkiMerchantSettings.requestMerchantSettings();
    }
}

File: app/src/main/java/com/kushkipagos/kushkidemo/services/RequestMerchantSettingsTask.kt
package com.kushkipagos.kushkidemo.services

import android.content.Context
import com.kushkipagos.exceptions.KushkiException
import com.kushkipagos.kushkidemo.contracts.AbstractRequestMerchantSettingsTask
import com.kushkipagos.models.MerchantSettings

class RequestMerchantSettingsTask: AbstractRequestMerchantSettingsTask {
    constructor(context: Context): super(context)

    @Throws(KushkiException::class)
    override fun requestMerchantSettings(): MerchantSettings {
        return _kushki.requestMerchantSettings()
    }
}